cli/tools/bundle/esbuild.rs

```diff
@@ -0,0 +1,101 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+use std::path::PathBuf;
+use std::sync::Arc;
+
+use deno_core::anyhow;
+use deno_core::anyhow::Context;
+use deno_core::error::AnyError;
+use deno_npm::npm_rc::ResolvedNpmRc;
+use deno_npm::registry::NpmRegistryApi;
+use deno_npm_cache::TarballCache;
+use deno_resolver::workspace::WorkspaceNpmPatchPackages;
+use deno_semver::package::PackageNv;
+
+use crate::cache::DenoDir;
+use crate::npm::CliNpmCache;
+use crate::npm::CliNpmCacheHttpClient;
+use crate::npm::CliNpmRegistryInfoProvider;
+use crate::sys::CliSys;
+
+pub const ESBUILD_VERSION: &str = "0.25.5";
```

Comment:
Should make it controllable via undocumented env var to make it easier to switch if needed?

Reply:
Or I guess this could be done by pointing to another already existing binary of esbuild